include/mbedtls/mbedtls_config.h

```diff
@@ -3280,7 +3280,8 @@
  * for a non-Aarch64 build it will be silently ignored.
  *
  * \note    Minimum compiler versions for this feature are Clang 4.0,
- * armclang 6.6 or GCC 6.0.
+ * armclang 6.6 or GCC 6.0. Targetting Thumb or 32-bit arm with GCC is not
```

Comment:
(minor, amusing) whether it's `targeting` or `targetting` is reasonably contested (https://english.stackexchange.com/a/205852/330378).

How about

```
Thumb or 32-bit Arm targets are not supported with GCC
```

Reply:
It does look wrong with two 't's.

Reply:
and reading the link I gave now I'm more properly awake "on both sides of the Atlantic, targeting, pivoting, parroting, etc., are normally spelled without -t doubling", so yes, `targeting` is preferred (although `preferred` is preferred to `prefered`)